concourse/pipeline.yml

```diff
@@ -249,8 +249,8 @@ jobs:
           HOSTNAME: gds-shielded-vulnerable-people-service-prod
           SECRET_KEY: ((svp-form/flask-secret-key-base-prod))
           NHS_OIDC_AUTHORITY_URL: https://auth.login.nhs.uk/
-          NHS_OIDC_LOGIN_CALLBACK_URL: https://coronavirus-shielding-support.service.gov.uk/nhs-login-callback
-          NHS_OIDC_REGISTRATION_CALLBACK_URL: https://coronavirus-shielding-support.service.gov.uk/nhs-registration-callback
+          NHS_OIDC_LOGIN_CALLBACK_URL: 	https://gds-shielded-vulnerable-people-service-prod.london.cloudapps.digital/nhs-login-callback
```

Comment:
Not sure about the whitespace here, I don't think it is significant, but it is visually jarring. Don't think it is blocker.